artemis-commons/src/test/java/org/apache/activemq/artemis/utils/ThreadLeakCheckRule.java

```diff
@@ -270,6 +270,18 @@ private boolean isExpectedThread(Thread thread) {
       } else if (threadName.contains("ObjectCleanerThread")) {
          // Required since upgrade to Netty 4.1.22 maybe because https://github.com/netty/netty/commit/739e70398ccb6b11ffa97c6b5f8d55e455a2165e
          return true;
+      } else if (threadName.contains("RMI TCP")) {
+         return true;
+      } else if (threadName.contains("RMI Scheduler")) {
+         return true;
+      } else if (threadName.contains("RMI RenewClean")) {
```

Comment:
Are you sure you're not hiding a thread leakage? can't you just stop the RMI Stub on this case?

Reply:
Nope, already checked in the first PR on the quorum vote